service/dap/server.go

```diff
@@ -433,6 +433,22 @@ func (s *Server) logToConsole(msg string) {
 }
 
 func (s *Server) onInitializeRequest(request *dap.InitializeRequest) {
+	if request.Arguments.PathFormat != "path" {
+		s.sendErrorResponse(request.Request, FailedToInitialize, "Failed to initialize",
+			fmt.Sprintf("Unsupported 'pathFormat' value '%v'.", request.Arguments.PathFormat))
```

Comment:
isn't %q not desirable?

Reply:
I updated to '%s' since request.Arguments.PathFormat is a string